doc/api/next_api_changes/behavior/26696-SR.rst

```diff
@@ -0,0 +1,4 @@
+The default value for the ``loc`` parameter of ``Cell`` has been changed from ``None`` to ``'right'``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The behavior does not change, since ``None`` was immediately replaced by ``'right'`` at initialization.
```

Comment:
The default changing is not the behaviour change. If you explicitly passed `loc=None`, it would break now, which _is_ the behaviour change.